sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java

```diff
@@ -694,15 +694,15 @@ private String validateAndGetDataSourceForIngest(final SqlInsert insert) throws
     }
 
     if (insert.getTargetColumnList() != null) {
-      throw new ValidationException("Ingestion with target column list is not supported.");
+      throw new ValidationException(insert.getOperator().getName() + " with target column list is not supported.");
```

Comment:
`insert.getOperator().getName()` is being used in several places.
Maybe assign it to a variable 
`final String insertOperatorName = insert.getOperator().getName();`

Reply:
Nice catch! Changed